### ISSUE TITLE
Change Row and Column Numbers

### ISSUE THREAD

**Issue by JessaWitzel** [JessaWitzel/battleship#3]
Rows should be structured as letters and columns should be real numbers, not indices.

**Comment by wwitzel3** [comment]
Agreed. We will want to handle this at user input time so that we can use standard indices for the internals of our program.

**Comment by wwitzel3** [comment]
I think we should also change all of the functions to accept the input in the order `row, col`. As for handling the user input format of `A, 7`. We can just have a helper method called `row_to_index` that would be easily implemented using `string.ascii_uppercase.index(A)`
